src/sentry/migrations/0241_auto__add_counter__add_unique_counter_project_ident__add_field_group_s.py

```diff
@@ -0,0 +1,609 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+from django.conf import settings
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'Counter'
+        db.create_table('sentry_projectcounter', (
+            ('id', self.gf('sentry.db.models.fields.bounded.BoundedBigAutoField')(primary_key=True)),
+            ('project', self.gf('sentry.db.models.fields.foreignkey.FlexibleForeignKey')(to=orm['sentry.Project'])),
+            ('ident', self.gf('django.db.models.fields.CharField')(max_length=40)),
+            ('value', self.gf('sentry.db.models.fields.bounded.BoundedPositiveIntegerField')()),
+        ))
+        db.send_create_signal('sentry', ['Counter'])
+
+        # Adding unique constraint on 'Counter', fields ['project', 'ident']
+        db.create_unique('sentry_projectcounter', ['project_id', 'ident'])
+
+        # Adding field 'Group.short_id'
+        db.add_column('sentry_groupedmessage', 'short_id',
+                      self.gf('sentry.db.models.fields.bounded.BoundedPositiveIntegerField')(null=True),
+                      keep_default=False)
+
+        # Adding unique constraint on 'Group', fields ['project', 'short_id']
+        db.create_unique('sentry_groupedmessage', ['project_id', 'short_id'])
```

Comment:
Since it's a `UNIQUE` constraint too, it'll also have to verify this with a full table scan to apply. I'm 99% sure we'll need to apply this index manually as `CONCURRENTLY`.